work_queue/src/work_queue.c

```diff
@@ -3934,7 +3934,7 @@ int work_queue_task_specify_file(struct work_queue_task *t, const char *local_na
 	// the worker(the worker on which the task will be executed) is unlikely to
 	// be known. Thus @param remote_name should not be an absolute path.
 	if(remote_name[0] == '/') {
-		fprintf(stderr, "Error: Remote name %s contains absolute path.\n", remote_name);
+		fatal("Error: Remote name %s contains absolute path.\n", remote_name);
```

Comment:
But totally correct. Thanks for that keen eyes!

On Wed, Jun 15, 2016 at 2:10 PM, Benjamin Tovar <notifications@github.com>
wrote:

> In work_queue/src/work_queue.c
> <https://github.com/cooperative-computing-lab/cctools/pull/1374#discussion_r67214858>
> :
>
> > @@ -3934,7 +3934,7 @@ int work_queue_task_specify_file(struct work_queue_task *t, const char *local_na
> >  	// the worker(the worker on which the task will be executed) is unlikely to
> >  	// be known. Thus @param remote_name should not be an absolute path.
> >  	if(remote_name[0] == '/') {
> > -		fprintf(stderr, "Error: Remote name %s contains absolute path.\n", remote_name);
> > +		fatal("Error: Remote name %s contains absolute path.\n", remote_name);
>
> Nitpick... Remote name '%s' is an absolute path.\n
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cooperative-computing-lab/cctools/pull/1374/files/78824be25c07eba40807cd98a62462dd0e6700c4#r67214858>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AE_SN5sE1r5eILPqaHnBYyi4oneXCbPAks5qMEAQgaJpZM4IwTzi>
> .
>